src/lib.rs

```diff
@@ -158,6 +150,21 @@
 //! on every call to `getrandom`, hence after the first successful call one
 //! can be reasonably confident that no errors will occur.
 //!
+//! # License
+//!
+//! The `getrandom` library is distributed under either of
+//!
+//! * [Apache License, Version 2.0][LICENSE-APACHE]
+//! * [MIT license][LICENSE-MIT]
+//!
+//! at your option.
+//!
+//! ## Contribution
+//!
+//! Unless you explicitly state otherwise, any contribution intentionally submitted
+//! for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
+//! dual licensed as above, without any additional terms or conditions.
```

Comment:
Good. (Though probably the contribution part only needs to appear in the README.)